formats/html_publication/publisher/details.json

```diff
@@ -21,6 +21,12 @@
     },
     "first_published_version": {
       "type": "boolean"
+    },
+    "isbn": {
+      "type": "string"
+    },
+    "print_meta_data_contact_address": {
```

Comment:
I've added descriptions to the schema. Please see here: https://github.com/alphagov/govuk-content-schemas/pull/561/files